schematron/v2.2.0/exemplary_files/L200_Audit.xml

```diff
@@ -929,6 +929,26 @@
                     <auc:UtilityID IDref="Utility-ElecGen" />
                   </auc:UtilityIDs>
                 </auc:ResourceUse>
+
+                <!-- submeter examples -->
+                <auc:ResourceUse ID="ResourceUse-Electricity-Lighting-Submeter">
+                  <auc:EnergyResource>Electricity</auc:EnergyResource>
+                  <auc:PercentResource>50</auc:PercentResource>
```

Comment:
@nllong this `auc:PercentResource` element seems really ugly / arbitrary.  I think we should either:

1. Clear up documentation.  Current documentation is: `Percentage of total consumption of this fuel type represented by this Resource Use record. (%)`. We should probably make this more explicit as an __estimate__ or we should provide documentation showing __how__ this element should be calculated.  For example, say we have 'meters' that look like:
```
Electricity Meter - EndUse = All end uses.  AnnualFuelUseNativeUnits = 100
|-- Electricity Meter - EndUse = Total lighting. AnnualFuelUseNativeUnits = 50
|-- Electricity Meter - EndUse = Heating. AnnualFuelUseNativeUnits = 50
```
Then, the `auc:PercentResource` for both the `Heating` and `Total lighting` `ResourceUse` elements should be `50`.  Basically saying that is calculated based on the __annual__ fuel use elements.

2. Remove this element, as it is redundant and confusing.

Reply:
Just looked back into the reason why we specified `auc:PercentResource` as a requirement.  So, in the 211 spreadsheet: `L2 - End Use Breakdown` sheet, it has the following:

![image](https://user-images.githubusercontent.com/28991233/96473144-452f6a00-11ff-11eb-895c-92b6b9722c59.png)

Basically, it wants you to enter in the estimated annual energy value, which it then uses to calculate a `% of total MJ` for each of the end uses.  @macintoshpie I think we should change the requirement instead of `auc:PercentResource` just to be `auc:AnnualFuelUseConsistentUnits`, `auc:AnnualFuelUseNativeUnits`, or both.  Thoughts?

Reply:
I think that sounds good, I'll make that change

Reply:
Consensus, require both:
- `auc:AnnualFuelUseConsistentUnits`
- `auc:AnnualFuelUseNativeUnits`

Don't require `auc:PercentResource`

Reply:
Addressed here: 2cbc6ac81ce20cb5af3ac4449a7f76a91560106e
Note that I had to update our method for calculating annual fuel use (checking imported/generated/etc energy sum) to only include ResourceUses linked to Utilities - these are our main meters (ie not submeters)

Reply:
🌭